README.md

```diff
@@ -1,29 +1,36 @@
-# React Demo App
+# Cognite Glimpse
 
-This repository shows how to bootstrap a [React]-based application at Cognite.
-It demonstrates best practices, such as:
+This repository is a showcase of how to build apps on top of [Cognite Data Fusion](https://cognite.com/cognite/cognite-data-fusion/developers/). This app is mainly useful for manufacturing operators, To perform the CNC machines as efficiently as possible and to understand when they are not performing at their best.
 
-- Lint setup to use
-- Jenkinsfile steps
-- TypeScript usage
-- Folder / component layout
-- ...
+This app is basically a widget container that shows the CNC machine's details. Widgets can be added, deleted (customized), and moved by dragging and dropping into an empty position.
 
-## Deploying the app
+## Getting started
 
-Please see the [deployment guide] for more information how to actually get this app into production.
-(It should be pretty easy!)
+- Clone the app
+  ```bash
+  $ git clone https://github.com/cognitedata/glimpse.git
+  ```
+- This app use [Firebase Cloud Firestore](https://firebase.google.com/docs/firestore) for saving data. You must [create a firebase project](https://firebase.google.com/docs/firestore/quickstart) and complete .env template inside the root folder with correct values. In the firestore create empty collections
 
-## Template repo
+  - app-config
+  - widget-config
 
-This repo is configured as a template, so it's easy to get started.
-Simply go to the [create repo page] and select "cognitedata/glimpse" as a template.
+- Install dependencies and start the app
+  ```bash
+  $ yarn && yarn start:https
+  ```
+  If you get an error about a certificate not being valid or trusted, then click "advanced" and "proceed" to skip.
 
-## Help
+## Unit testing
 
-If you have any questions, please join us in [#frontend] and ask away!
+We are using [react-testing-library](https://testing-library.com/docs/react-testing-library/intro) with [jest](https://jestjs.io/) as the test runner. It is prefered to write tests that closely resemble how your web pages are used. For more info read [here](https://testing-library.com/docs/guiding-principles).
 
-[react]: https://reactjs.org/
-[deployment guide]: https://cognitedata.atlassian.net/wiki/spaces/FAS/pages/1003225162/How+to+deploy+on+Frontend+App+Server+FAS
-[#frontend]: https://cognitedata.slack.com/archives/C6KNJCEEA
-[create repo page]: https://github.com/organizations/cognitedata/repositories/new
+Run all tests
+
+```bash
+$ yarn test
+```
+
+## License
+
+[Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
We need to add some details about the design of the app.  Lets do that incrementally.